### ISSUE TITLE
ETA V4.0.0 Release

### ISSUE THREAD

**Issue by raupie** [SimonPadbury/b4st#47]
Any eta for using the new V4.0.0 release?

**Comment by SimonPadbury** [comment]
Yes, this week. Also, other improvements for child theming. A few hours more, so maybe by Friday morning UK time. Fixed a problem with Font Awesome 5 pseudo css too. With test with newest jQuery at the same time.

**Comment by raupie** [comment]
Awesome. Thanks for keeping this current. Best WP BS theme I've found!

**Comment by SimonPadbury** [comment]
Hi Travis,

Almost ready. Can you “kick the tires” on this shiny new b4st 2.0 before I sync on GitHub tomorrow?

https://www.dropbox.com/s/cjb1umjr7gb2nix/b4st.zip?dl=0 <https://www.dropbox.com/s/cjb1umjr7gb2nix/b4st.zip?dl=0>

Simon

**Comment by raupie** [comment]
Yup, I'll spin up a container and try it out!

**Comment by raupie** [comment]
Got it up and running in no time. copy/pasted a few bootstrap examples from the documentation and it appeared as expected.

**Comment by SimonPadbury** [comment]
Thankyou — is here anything you would want to see changed? Or anything majorly wrong?

**Comment by liammday** [comment]
Have you considered adding the easing/scrollspy feature I expanded on in the other issue thread?

**Comment by SimonPadbury** [comment]
Sorry for the delay. I hope to launch b4st 2.0 tomorrow. Then I will look into scrollspy

**Comment by raupie** [comment]
I honestly like that it's as plain and generic as possible. I haven't used v4 enough to make any recommendations but if something sticks out I won't keep it to myself.

**Comment by SimonPadbury** [comment]
b4st v2.0-beta now online.

I plan on making a few changes next week, when I have solved a couple of issues.

**Comment by raupie** [comment]
Any recommendations for an upgrade procedure?

**Comment by SimonPadbury** [comment]
Not sure what you mean.
